source/help/messaging/organizing-conversations.rst

```diff
@@ -0,0 +1,72 @@
+Organizing Conversations using Collapsed Reply Threads (Beta)
+=============================================================
+
+Threads are a key part of the messaging experience in Mattermost. Threads organize conversations within channels, and threads enable users to discuss topics without adding noise to channels or Direct Messages. 
+
+Collapsed Reply Threads offers an enhanced experience for users participating in conversations by:
+
+- Improving channel details to find, follow, and resume conversations more easily.
+- Prioritizing how conversation threads are displayed in Mattermost.
+
+.. image:: ../../images/collapsed-reply-threads.gif
+  :alt: Organize conversations using Collapsed Reply Threads.
+
+Your System Admin must enable the Collapsed Reply Threads feature in the System Console. See our `Configuration Settings <https://docs.mattermost.com/administration/config-settings.html#collapsed-reply-threads-beta>`__ documentation for details. You may additionally need to enable this feature for your Mattermost account by going to **Account Settings > Display > Collapsed Reply Threads (Beta)**.
+
+Known Issues
+------------
+
+.. important::
+
+    Collapsed Reply Threads is available as a beta release from Mattermost v5.37. You may experience minor issues with this beta functionality. 
+
+You may encounter the following known issues after enabling Collapsed Reply Threads:
+
+- You may automatically follow a large number of historical threads resulting in a large number of unread threads. To resolve this, go to  **Threads** in the channel sidebar, then select **Mark All As Read**. Older threads you’re automatically following will continue to notify you of new replies.
```

Comment:
Let's please carbon copy this section from the proposed blog post: https://mattermost.atlassian.net/wiki/spaces/Threads/blog/2021/06/25/2031288323/Collapsed+Reply+Threads+are+here+Get+early+access+to+the+beta
![image](https://user-images.githubusercontent.com/14333569/123670968-9eba5080-d7f2-11eb-803a-728ff00fb412.png)


Minus this line : "These issues and available workarounds are shared in our documentation." ;)

Reply:
@amyblais please add a link to this section of this doc (Known issues) to the important upgrade notes. You can also link to this blog post that will go live mid-July: https://mattermost.com/blog/collapsed-reply-threads-beta/